### PR TITLE
Fix header to show 'Company Name' at the middle of the topbar and fix invoice config

### PR DESCRIPTION
**Pull request by RamkrishnaMondal**

### CODE REVIEW

File: application/views/partial/header.php [hunk 1]
```diff
@@ -97,9 +97,14 @@
 				</div>
 				
 				<div class="navbar-right" style="margin:0">
-					<?php echo $this->config->item('company') . "  |  $user_info->first_name $user_info->last_name  |  " . ($this->input->get("debug") == "true" ? $this->session->userdata('session_sha1') : ""); ?>
+					<?php echo $this->config->item('common_welcome') . "    $user_info->first_name $user_info->last_name  |  " . ($this->input->get("debug") == "true" ? $this->session->userdata('session_sha1') : ""); ?>
```
Comment (daN4cat) [inline application/views/partial/header.php]: I would remove the common_welcome so you avoid issues when the screen is resized (e.g. running OSPOS on mobile or tablet).

Comment (RamkrishnaMondal) [inline application/views/partial/header.php]: Ok.

Comment (daN4cat) [inline application/views/partial/header.php]: thanks

Comment (RamkrishnaMondal) [inline application/views/partial/header.php]: Done

**Comment by daN4cat** [comment]
Merged

**Comment by RamkrishnaMondal** [comment]
Thanks!

**Pull request merged**